frame/nfts/src/features/attributes.rs

```diff
@@ -159,6 +202,15 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok(())
 	}
 
+	/// Sets multiple attributes for an item or a collection.
+	///
+	/// This function checks the pre-signed data is valid and updates the attributes of an item or
+	/// collection. It is limited by [`T::MaxAttributesPerCall`] to prevent excessive storage
```

Comment:
```suggestion
	/// collection. It is limited by [`T::MaxAttributesPerCall`](https://docs.rs/pallet-nfts/latest/pallet_nfts/pallet/trait.Config.html#associatedtype.MaxAttributesPerCall)
	/// to prevent excessive storage
```
Because T is generic I think we need an explicit link here otherwise cargo doc emits a warning that T isn't defined.